batch7crud-roll5/src/main/java/com/lftechnology/batch7crud/db/DBConnection.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * 
+ */
+package com.lftechnology.batch7crud.db;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import javax.sql.DataSource;
+
+import com.lftechnology.batch7crud.exception.DataException;
+
+/**
+ * @author leapfrog
+ *
+ */
+public class DBConnection {
+
+    public static Connection getConnection() throws DataException {
+        try {
+            Context initContext = new InitialContext();
+            Context envContext = (Context) initContext.lookup("java:/comp/env");
+            DataSource ds = (DataSource) envContext.lookup("jdbc/EmployeeManagement");
+            return ds.getConnection();
+      
+        } catch (SQLException e) {
+            throw new DataException(e.getMessage());
+
+        } catch (NamingException e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1166)